src/components/FilterChip/FilterChip.stories.tsx

```diff
@@ -16,5 +16,6 @@ export const filterValue: Story = {
     filter: "Filter: ",
     value: "Value",
     addView: false,
+    onRemove: () => console.log("test")
```

Comment:
Use actions for action related functions being called so we can leverage storybook's action tab.

At top of file:
`import { action } from '@storybook/addon-actions';`

```suggestion
    onRemove: () => action('onRemove')
```

more on it here: https://storybook.js.org/docs/essentials/actions

Reply:
change made